pkg/kubelet/kubelet.go

```diff
@@ -2922,7 +2922,7 @@ func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod, podStatus *kubecontaine
 	if fit {
 		// Update pod resource allocation checkpoint
 		if err := kl.statusManager.SetPodAllocation(pod); err != nil {
-			return nil, err
+			klog.ErrorS(err, "SetPodAllocation failed", "pod", klog.KObj(pod))
```

Comment:
I changed my mind about this change. I think we still want to fail the resize if the allocation can't be checkpointed, otherwise the node could become unstable on restart. Let's revert this line to what we had before.